src/main/java/io/vertx/amqp/impl/AmqpConnectionImpl.java

```diff
@@ -39,12 +39,14 @@
   public static final String PRODUCT = "vertx-amqp-client";
   public static final Symbol PRODUCT_KEY = Symbol.valueOf("product");
 
-  public static final UnsignedLong NO_LOCAL_CODE = UnsignedLong.valueOf(0x0000_468C_0000_0003L);
-  public static final Symbol NO_LOCAL_NAME = Symbol.valueOf("apache.org:no-local-filter:list");
-  public static final DescribedType NO_LOCAL_TYPE = new UnknownDescribedType(NO_LOCAL_CODE, "NoLocalFilter{}");
+  // For the "apache.org:no-local-filter:list" filter
+  private static final Symbol NO_LOCAL_KEY = Symbol.valueOf("no-local");
+  private static final UnsignedLong NO_LOCAL_DESCRIPTOR = UnsignedLong.valueOf(0x0000_468C_0000_0003L);
+  private static final DescribedType NO_LOCAL_FILTER = new UnknownDescribedType(NO_LOCAL_DESCRIPTOR, "NoLocalFilter{}");
 
-  public static final UnsignedLong SELECTOR_CODE = UnsignedLong.valueOf(0x0000_468C_0000_0004L);
-  public static final Symbol SELECTOR_NAME = Symbol.valueOf("apache.org:selector-filter:string");
+  // For the "apache.org:selector-filter:string" filter
+  private static final Symbol SELECTOR_KEY = Symbol.valueOf("selector");
```

Comment:
Out of curiosity, why are you using "selector" instead of "apache.org:selector-filter:string" as mentioned on https://www.amqp.org/specification/1.0/filters here?

Reply:
Because that is the filters symbolic descriptor, not the value that has to be used as the filter map key (which is actually undefined and not meant to influence behaviour).

Reply:
i.e you can send apache.org:selector-filter:string as the described type descriptor rather than the ulong numeric descriptor

Reply:
ok, thank you!